perception/lidar_centerpoint/README.md

```diff
@@ -234,35 +222,16 @@ You can find it in `mmdetection3d/projects/AutowareCenterPoint` file.
 python projects/AutowareCenterPoint/centerpoint_onnx_converter.py --cfg projects/AutowareCenterPoint/configs/centerpoint_custom.py --ckpt work_dirs/centerpoint_custom/YOUR_BEST_MODEL.pth --work-dir ./work_dirs/onnx_models
 ```
 
-#### Create the config file for the custom model
-
-Create a new config file named **centerpoint_custom.param.yaml** under the config file directory of the lidar_centerpoint node. Sets the parameters of the config file like
-point_cloud_range, point_feature_size, voxel_size, etc. according to the training config file.
-
-```yaml
-/**:
-  ros__parameters:
-    class_names: ["CAR", "TRUCK", "BUS", "BICYCLE", "PEDESTRIAN"]
-    point_feature_size: 4
-    max_voxel_size: 40000
-    point_cloud_range: [-51.2, -51.2, -3.0, 51.2, 51.2, 5.0]
-    voxel_size: [0.2, 0.2, 8.0]
-    downsample_factor: 1
-    encoder_in_feature_size: 9
-    # post-process params
-    circle_nms_dist_threshold: 0.5
-    iou_nms_target_class_names: ["CAR"]
-    iou_nms_search_distance_2d: 10.0
-    iou_nms_threshold: 0.1
-    yaw_norm_thresholds: [0.3, 0.3, 0.3, 0.3, 0.0]
-```
+#### Adjust the config file for the custom model
+
+All the ROS parameters have been moved into `.param.yaml` files, please set the parameters of the config file like point*cloud_range, point_feature_size, voxel_size, etc. in the config files. \*\*centerpoint*(your_selection_of_model).param.yaml\*\* files are under the config file directory of the lidar_centerpoint node. The information for these parameters are shown in the [Parameters](#parameters) section.
```

Comment:
Sorry, this lacks a little in both writing and content.
The idea of the section is to describe the process of creating the files for a new model. In addition to some grammatical errors, instead of giving examples of parameters of the config file explain what kind of parameters they have. This links to the the ml config file, which is not mentioned here

Reply:
> Sorry, this lacks a little in both writing and content. The idea of the section is to describe the process of creating the files for a new model. In addition to some grammatical errors, instead of giving examples of parameters of the config file explain what kind of parameters they have. This links to the the ml config file, which is not mentioned here

@knzo25 Thank you for your instructions, I have updated the README.md file, please check if this meets your expectation. If you can give proposal for me to accept, it would be of great help for me. I will click the button commit suggestion afterwards. I really learned a lot with your help for this PR, thank you so much.